src/helpers/typeHelper.js

```diff
@@ -1,9 +1,3 @@
-import DocumentDetail from '@/components/results/detail/DocumentDetail';
```

Comment:
This caused an import cycle, so I moved it to the detail view, this is the only usage of it.